routing/routing.go

```diff
@@ -62,10 +62,10 @@ func NewRouting(routes []APIRoute, store stores.Store, config *config.Config) *A
 }
 
 var ApiRoutes = []APIRoute{
-	{"services:create", "POST", "/services", handlers.ServiceCreate},
-	{"services:ListOne", "GET", "/services/{name}", handlers.ServiceListOne},
-	{"services:ListAll", "GET", "/services", handlers.ServiceListAll},
-	{"authMethod:ListOne", "GET", "/services/{service}/hosts/{host}/authM", handlers.AuthMethodListOne},
+	{"serviceTypes:create", "POST", "/service-types", handlers.ServiceTypeCreate},
```

Comment:
Can the path refactoring be reflected in handlers tests also for consistency?

Reply:
yeah, fixing it now.